tools/weekeventregconvert.py

```diff
@@ -279,10 +279,10 @@
     (34 << 8) | 0x01: "WEEKEVENTREG_34_01",
     (34 << 8) | 0x02: "WEEKEVENTREG_34_02",
     (34 << 8) | 0x04: "WEEKEVENTREG_34_04",
-    (34 << 8) | 0x08: "WEEKEVENTREG_34_08",
+    (34 << 8) | 0x08: "WEEKEVENTREG_SWAMP_SPIDERHOUSE_TALKED",
```

Comment:
```suggestion
    (34 << 8) | 0x08: "WEEKEVENTREG_SWAMP_SPIDER_HOUSE_TALKED",
```
Two words for consistency